kernel/process/process.rs

```diff
@@ -207,6 +207,11 @@ impl Process {
         self.pid
     }
 
+    /// The thread ID
```

Comment:
```suggestion
    /// The thread ID.
```

Please add a fullstop.